en/04_Changelogs/5.0.0.md

```diff
@@ -26,9 +34,65 @@ title: 5.0.0 (unreleased)
   - [Other features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the [list Commercially Supported Modules for Silverstripe CMS 5](/Project_Governance/Supported_Modules).
+
+### Module not supported in Silverstripe CMS 5
```

Comment:
Header got renamed to "Community supported in Silverstripe CMS 5"

Reply:
Please add the word "modules" in there

Reply:
Hmmmm I don't think this rename matches what the section is about. It's really about modules which _used_ to be commercially supported. This heading makes me think it's going to talk about my random module which silverstripe doesn't know exists.

Please change this back - or otherwise keep it clearly on subject (e.g. "Previously commercially supported modules" or "Dropping commercial support for some modules")